js/_codemirror.ts

```diff
@@ -64,6 +66,7 @@ export const extensions = {
             { key: 'Enter', run: insertNewline },
             { key: 'Tab',   run: insertTab },
         ]),
+        drawSelection(),
```

Comment:
In vim mode, this is needed to show selections at all.

It could be limited in default mode, if that's preferable. I thought it actually has a slight aesthetic improvement (see [the docs](https://codemirror.net/docs/ref/#view.drawSelection) for more details about the tradeoff being made by enabling this).

### Example without in normal mode
![Screen Shot 2022-08-05 at 7 03 57 PM](https://user-images.githubusercontent.com/11131775/183222139-d751c1af-c4b0-4334-b67e-308bf5ced6c3.png)

### With the extension in normal mode
![Screen Shot 2022-08-05 at 7 01 42 PM](https://user-images.githubusercontent.com/11131775/183222117-5c612cf2-fa02-46d0-b870-5666d752f075.png)